test/trento/activity_logging/metadata_enricher_test.exs

```diff
@@ -219,28 +219,32 @@ defmodule Trento.ActivityLog.MetadataEnricherTest do
   end
 
   describe "enriching operation activities" do
-    test "should enrich operation completed events in hosts" do
-      %{id: host_id, hostname: hostname} = insert(:host)
+    for saptune_operation <- [:saptune_solution_apply, :saptune_solution_change] do
+      @saptune_operation saptune_operation
```

Comment:
nitpick:
as this looks a generic test, maybe we can rename `saptune_operation` to `operation/current_operatrion`

Reply:
13c6ee8c722373a1d1f0a246246eb656f14fb2b2